data/topic-schema/src/main/resources/net/corda/schema/P2P.yaml

```diff
@@ -15,6 +15,12 @@ topics:
       - flow
       - membership
     config:
+  P2POutDqlTopic:
```

Comment:
Should this be P2POut**Dlq**Topic rather than P2POut**Dql**Topic ?

Reply:
Updated, thanks.